tests/test_class_loading.py

```diff
@@ -137,3 +144,70 @@ def test_find_test_module_of_class(mock_index):
     path = index.find_test_module_of_class("TestFixture")
     expected = "tests.base_tests.class_loading"
     assert path == expected
+
+
+##
+# Cloudy side tests
+
+
+def test_try_load_not_existing_class(pytester):
```

Comment:
[use try + gerund here](https://learnenglish.britishcouncil.org/grammar/intermediate-to-upper-intermediate/verbs-followed-by-ing-or-by-to-infinitive-2): `try_loading`

Reply:
Or maybe you can even leave out the `try` altogether:
`test_load_nonexisting_module` (same for the other new test methods)

Reply:
Good input, thanks!